ext/web/16_report_error.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
I think you could safely merge this into `02_event.js` and get rid of the circular problem of storing `__bootstrap` namespace. WDYT?